### PR TITLE
Merging DjangoCon workshop

### PR DESCRIPTION
**Pull request by Lotram**

### CODE REVIEW

File: notebooks/10-optimization-tips/queries.ipynb
Comment (phililippe) [inline notebooks/10-optimization-tips/queries.ipynb:1]: You used the structure `djangocon_workshop/notebooks`, so you can move this file to `10-optimization-tips/notebooks/`

File: playground/settings.py [hunk 1]
```diff
@@ -23,12 +26,14 @@
...
-SECRET_KEY = os.environ.get("DJANGO_SECRET_KEY", "django-insecure - replace by your key")
+SECRET_KEY = os.environ.get(
+    "DJANGO_SECRET_KEY", "django-insecure - replace by your key"
+)
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = True
 
-ALLOWED_HOSTS = []
+ALLOWED_HOSTS = ["testserver", "localhost"]
```
Comment (phililippe) [inline playground/settings.py:36]: Is it required for a repo like this one? Should not cause issue though (as long it is not run from Kube)

File: README.md [hunk 2]
```diff
@@ -50,38 +51,26 @@ docker compose up -d
...
-This notebook is based on the `books` Django Application that includes two Models:
-
-- Book
-- Person
-
-A Person can be either the `author` of a Book or one of its `readers`.
-All books written by a Person can be accessed through the `writings` related name of the `author` ForeignKey.
-The `readers` relation is defined as a ManyToManyField, accessible from Person using `writings` related name.
-
-```mermaid
```
Comment (phililippe) [inline README.md:64]: I think providing context on the app is useful. Perhaps can you provide a link to a Slide from your workshop if you think mermaid is not adapted?

File: README.md [hunk 3]
```diff
@@ -50,38 +51,26 @@ docker compose up -d
...
+### Load DB dump
 
 ```
+cat db_dump/dbdump*.db.part > db_dump/dumpdir.tar.gz
+mkdir -p dumpdir
+pigz -dc db_dump/dumpdir.tar.gz | tar -C dumpdir --strip-components 1 -xf -
+pg_restore -j 10 -Fd -O -h localhost -p $DATABASE_PORT -U $DATABASE_USER -d $DATABASE_NAME dumpdir
+```
+
+## DjangCon
```
Comment (phililippe) [inline README.md:73]: You should also provide a link to 10-django-tips. The article is still online so people may come for this content. It's good if they see there's additional (and more complete!) content, but we should still mention the one they first come for.

File: playground/settings.py [hunk 4]
```diff
@@ -149,3 +154,21 @@
 # https://docs.djangoproject.com/en/4.1/ref/settings/#default-auto-field
 
 DEFAULT_AUTO_FIELD = "django.db.models.BigAutoField"
+
+
+SHELL_PLUS_PRINT_SQL_TRUNCATE = None
+SHELL_PLUS_PRINT_SQL = True  # TODO Change back to false to start
+
+
+if SENTRY_DSN := os.getenv("SENTRY_DSN"):
```
Comment (phililippe) [inline playground/settings.py:163]: To add to .env.example ?

File: README.md (hunk 3 above)
Comment (Lotram) [reply README.md:73]: It's already at the top of the file :)

**Comment by Lotram** [comment]
@phililippe  I took your remarks into account, removed a bunch of generated files, and squashed everything

**Review by phililippe — APPROVED** [review]

**Pull request merged**